discovery-provider/alembic/trigger_sql/handle_challenge_disbursements.sql

```diff
@@ -8,14 +8,15 @@ begin
   if reward_manager_tx is not null then
 	  -- create a notification for the challenge disbursement
 	  insert into notification
-		(slot, user_ids, timestamp, type, specifier, data)
+		(slot, user_ids, timestamp, type, group_id, specifier, data)
```

Comment:
Bit confused about the meaning of the `group_id` and `specifier` field names.

Is meaning of `specifier` same as before (i.e. `challenge_reward:1:mobile_install`) and `group_id` is the extra info for the unique key?

Either way... the field ordering looks inconsistent across the diff... which we'll have to fix regardless of naming convention haha.

Reply:
group_id is the column I want to eventually group the notifications by, so in this case it's `challegne_reward:1:mobile_install` because the challenge disbursements shouldn't ever be grouped - for this type, it should be unique because the challenge, but can make it more specific